spylon_kernel/scala_kernel.py

```diff
@@ -85,14 +142,47 @@ def do_execute_direct(self, code, silent=False):
             return self.Error(e.scala_message)
 
     def get_completions(self, info):
+        """Gets completions from the kernel based on the provided info.
+
+        Part of the MetaKernel API.
+
+        Parameters
+        ----------
+        info : dict
+            Information returned by `metakernel.parser.Parser.parse_code`
+            including `code`, `help_pos`, `start`, etc.
+
+        Returns
+        -------
+        list of str
+            Possible completions for the code
+        """
         return self._scalamagic.get_completions(info)
 
     def get_kernel_help_on(self, info, level=0, none_on_fail=False):
```

Comment:
As discussed in person, will doc how these are all implementing the expected MetaKernel interface.